libmscore/textlinebase.cpp

```diff
@@ -106,11 +106,34 @@ void TextLineBaseSegment::draw(QPainter* painter) const
             textlineLineWidth *= mag();
       QPen pen(color, textlineLineWidth, tl->lineStyle());
       QPen solidPen(color, textlineLineWidth, Qt::SolidLine);
-      if (tl->lineStyle() == Qt::CustomDashLine) {
-            QVector<qreal> dashes { tl->dashLineLen(), tl->dashGapLen() };
-            pen.setDashPattern(dashes);
+
+      //Replace generic Qt dash patterns with improved equivalents to show true dots
+      QVector<qreal> dotted        = { 0.01, 1.99 }; // tighter than Qt Dotline equivalent { 0.01, 2.99 }
+      QVector<qreal> dashed        = { 3.0, 3.0 };
```

Comment:
see above, compensating for SquareCap?

Reply:
same. I better add a comment there // Qt DashLine is { 4.0, 2.0 }